### PR TITLE
fix(collaborator):fixed tests

### PR DESCRIPTION
**Pull request by cnlangzi**
## Fixes
- fixed https://github.com/0chain/blobber/issues/573

## PRs
- blobber: https://github.com/0chain/blobber/pull/613

### CODE REVIEW

**Comment by service-0chain** [comment]
Manual system tests [failure] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/collaborator_update  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | fix/collaborator_update  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

**Comment by service-0chain** [comment]
Manual system tests [success] with the following config
| config  | value |
| ------------- | ------------- |
| tests skipped  | FALSE  |
| system-tests  | fix/collaborator_update  |
| gosdk  | NOT OVERRIDDEN  |
| zboxcli  | staging  |
| zwalletcli  | staging  |
| 0chain  | staging  |
| blobber  | fix/collaborator_update  |
| authorizer  | staging  |
| 0box  | staging  |
| 0block  | staging  |
| 0dns  | staging  |
| explorer  | staging  |
| 0proxy  | staging  |
| 0search  | staging  |

File: tests/cli_tests/zboxcli_collaborator_system_test.go [hunk 1]
```diff
@@ -644,19 +644,19 @@ func TestCollaborator(t *testing.T) {
...
+		updatedLocalPath := generateRandomTestFileName(t)
+		err = createFileWithSize(updatedLocalPath, 128*KB)
 		require.Nil(t, err)
 
 		output, err = updateFileWithWallet(t, collaboratorWalletName, configPath, map[string]interface{}{
 			"allocation": allocationID,
 			"remotepath": remotepath,
-			"localpath":  localpath,
+			"localpath":  updatedLocalPath,
 		}, false)
```
Comment (Kishan-Dhakan) [inline tests/cli_tests/zboxcli_collaborator_system_test.go:655]: Please turn this to true as now it is not expected to fail

**Review by Kishan-Dhakan — CHANGES_REQUESTED** [review]

File: tests/cli_tests/zboxcli_collaborator_system_test.go (hunk 1 above)
Comment (cnlangzi) [reply tests/cli_tests/zboxcli_collaborator_system_test.go]: fixed

**Review by peterlimg — APPROVED** [review]
lgtm

**Pull request merged**